library/src/main/java/org/xmtp/android/library/Group.kt

```diff
@@ -337,6 +352,14 @@ class Group(
         )
     }
 
+    suspend fun updateMessageExpirationPermission(newPermissionOption: PermissionOption) {
+        return libXMTPGroup.updatePermissionPolicy(
+            FfiPermissionUpdateType.UPDATE_METADATA,
+            PermissionOption.toFfiPermissionPolicy(newPermissionOption),
+            FfiMetadataField.MESSAGE_EXPIRATION
```

Comment:
Need this from libxmtp

Reply:
Since we set the messageDisappearing setting always be admin, I don't think we need this, do we?